internal/providers/terraform/aws/mq_broker.go

```diff
@@ -0,0 +1,105 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetMQBrokerRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_mq_broker",
+		RFunc: NewMQBroker,
+	}
+}
+
+func NewMQBroker(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := d.Get("region").String()
+	engine := d.Get("engine_type").String()
+	instanceType := d.Get("host_instance_type").String()
+	deploymentMode := d.Get("deployment_mode").String()
+	isActiveStandby := false
+	if deploymentMode == "ACTIVE_STANDBY_MULTI_AZ" {
+		isActiveStandby = true
+	}
+
+	nodesCount := decimalPtr(decimal.NewFromInt(1))
+	if u != nil && u.Get("nodes_count").Exists() {
+		nodesCount = decimalPtr(decimal.NewFromInt(u.Get("nodes_count").Int()))
+	}
+	var storageSizeGB *decimal.Decimal
+	if u != nil && u.Get("storage_size_gb").Exists() {
+		storageSizeGB = decimalPtr(decimal.NewFromInt(u.Get("storage_size_gb").Int()))
+	}
+
+	return &schema.Resource{
+		Name: d.Address,
+		CostComponents: []*schema.CostComponent{
+			instance(region, engine, instanceType, isActiveStandby, nodesCount),
+			storage(region, engine, isActiveStandby, storageSizeGB, nodesCount),
+		},
+	}
+}
+
+func instance(region, engine, instanceType string, isActiveStandby bool, nodesCount *decimal.Decimal) *schema.CostComponent {
+	deploymentOption := "Single-AZ"
+	if isActiveStandby {
+		deploymentOption = "Multi-AZ"
+	}
+	return &schema.CostComponent{
+		Name:           fmt.Sprintf("Instance (%s, %s)", engine, instanceType),
+		Unit:           "hours",
+		UnitMultiplier: 1,
+		HourlyQuantity: nodesCount,
+		ProductFilter: &schema.ProductFilter{
+			VendorName:    strPtr("aws"),
+			Region:        strPtr(region),
+			Service:       strPtr("AmazonMQ"),
+			ProductFamily: strPtr("Broker Instances"),
+			AttributeFilters: []*schema.AttributeFilter{
+				{Key: "usagetype", ValueRegex: strPtr(fmt.Sprintf("/%s/", instanceType))},
+				{Key: "brokerEngine", Value: strPtr(engine)},
+				{Key: "deploymentOption", Value: strPtr(deploymentOption)},
+			},
+		},
+	}
+}
+
+func storage(region, engine string, isActiveStandby bool, storageSizeGB, nodesCount *decimal.Decimal) *schema.CostComponent {
+	if storageSizeGB != nil {
+		freeSizeGB := decimal.NewFromInt(5)
```

Comment:
This only applies for EFS, and it's not clear if the first 5GB is free for all instances in the first cluster, or just the first instance, or just the first AmazonMQ cluster that's created in an AWS account... So I wonder if it's better to just remove this?